docs/reference/commandline/service_create.md

```diff
@@ -384,9 +402,24 @@ volumes in a service:
      </p>
     </td>
   </tr>
+  <tr>
+    <td><b>bind-nonrecursive</b></td>
+    <td>
+      By default, submounts are recursively bind-mounted as well. However, this behavior can be confusing when a
+      bind mount is configured with <tt>readonly</tt> option, because submounts are not mounted as read-only.
+      Set <tt>bind-nonrecursive</tt> to disable recursive bind-mount.<br />
+
+      A value is optional:
+
+      <ul>
+        <li><tt>true</tt> or <tt>1</tt>: Disables recursive bind-mount.</li>
+        <li><tt>false</tt> or <tt>0</tt>: Default if you do not provide a value. Enables recursive bind-mount.</li>
+      </ul>
+    </td>
+  </tr>
 </table>
 
-#### Bind Propagation
+##### Bind Propagation
```

Comment:
Extra `#`?

Reply:
intentional

Reply:
(small nit) the docs use sentence-case, so lowercase `propagation`

```suggestion
##### Bind propagation
```

Reply:
Here's a diff with all the changes;

```patch
diff --git a/docs/reference/commandline/service_create.md b/docs/reference/commandline/service_create.md
index cf057bef..91aba2da 100644
--- a/docs/reference/commandline/service_create.md
+++ b/docs/reference/commandline/service_create.md
@@ -386,7 +386,7 @@ The following options can only be used for bind mounts (`type=bind`):
   <tr>
     <td><b>bind-propagation</b></td>
     <td>
-      <p>See [Bind Propagation section](#bind-propagation).</p>
+      <p>See the <a href="#bind-propagation">bind propagation section</a>.</p>
     </td>
   </tr>
   <tr>
@@ -408,9 +408,9 @@ The following options can only be used for bind mounts (`type=bind`):
       By default, submounts are recursively bind-mounted as well. However, this behavior can be confusing when a
       bind mount is configured with <tt>readonly</tt> option, because submounts are not mounted as read-only.
       Set <tt>bind-nonrecursive</tt> to disable recursive bind-mount.<br />
-
-      A value is optional:
-
+      <br />
+      A value is optional:<br />
+      <br />
       <ul>
         <li><tt>true</tt> or <tt>1</tt>: Disables recursive bind-mount.</li>
         <li><tt>false</tt> or <tt>0</tt>: Default if you do not provide a value. Enables recursive bind-mount.</li>
@@ -419,7 +419,7 @@ The following options can only be used for bind mounts (`type=bind`):
   </tr>
 </table>
 
-##### Bind Propagation
+##### Bind propagation
 
 Bind propagation refers to whether or not mounts created within a given
 bind mount or named volume can be propagated to replicas of that mount. Consider
@@ -456,7 +456,7 @@ volumes do not support bind propagation.
 For more information about bind propagation, see the
 [Linux kernel documentation for shared subtree](https://www.kernel.org/doc/Documentation/filesystems/sharedsubtree.txt).
 
-#### Options for Named Volumes
+#### Options for named volumes
 
 The following options can only be used for named volumes (`type=volume`):
 
@@ -492,9 +492,9 @@ The following options can only be used for named volumes (`type=volume`):
       the Engine copies those files and directories into the volume, allowing
       the host to access them. Set <tt>volume-nocopy</tt> to disable copying files
       from the container's filesystem to the volume and mount the empty volume.<br />
-
-      A value is optional:
-
+      <br />
+      A value is optional:<br />
+      <br />
       <ul>
         <li><tt>true</tt> or <tt>1</tt>: Default if you do not provide a value. Disables copying.</li>
         <li><tt>false</tt> or <tt>0</tt>: Enables copying.</li>
```